it/controllers/benefits/reimbursed/VouchersOrCreditCardsBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,534 @@
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  ".show" should {
+
+    import Selectors._
+
+    userScenarios.foreach { user =>
+
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+
+        "render 'vouchers or credit cards' page with the correct content with no pre-filling" which {
+          lazy val result: WSResponse = {
+            dropEmploymentDB()
+            userDataStub(userData(fullEmploymentsModel()), nino, taxYearEOY)
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("employerName", hmrc = true,
+              benefits = Some(benefits(fullReimbursedCostsVouchersAndNonCashModel.copy(vouchersAndCreditCardsQuestion = None))))), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(pageUrl(taxYearEOY), welsh = user.isWelsh,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          textOnPageCheck(expectedCaption, captionSelector)
+          textOnPageCheck(user.specificExpectedResults.get.paragraph, paragraphSelector)
+          radioButtonCheck(yesText, 1, Some(false))
+          radioButtonCheck(noText, 2, Some(false))
+          buttonCheck(expectedButtonText, continueButtonSelector)
+          formPostLinkCheck(continueLink, continueButtonFormSelector)
+          welshToggleCheck(user.isWelsh)
+        }
+
+        "render 'vouchers or credit cards' page with the correct content with yes pre-filled" which {
+          lazy val result: WSResponse = {
+            dropEmploymentDB()
+            userDataStub(userData(fullEmploymentsModel()), nino, taxYearEOY)
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("employerName", hmrc = true,
+              benefits = Some(benefits(fullReimbursedCostsVouchersAndNonCashModel)))), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(pageUrl(taxYearEOY), welsh = user.isWelsh,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          textOnPageCheck(expectedCaption, captionSelector)
+          textOnPageCheck(user.specificExpectedResults.get.paragraph, paragraphSelector)
```

Comment:
need to add textOnPage checks for the bullet pointed text

Reply:
Updated as requested.